### ISSUE TITLE
Enable permissions to use location in the sample applications

### ISSUE THREAD

**Issue by anttikajanus** [Esri/arcgis-runtime-samples-xamarin#100]
Add permissions to access the emulator and device locations to all applications. 

On iOS add info.plist manifest
````
  <key>NSLocationWhenInUseUsageDescription</key>
  <string>You are about to use location!</string>
```` 

On Android add AssemblyInfo.cs manifest
````
[assembly: UsesPermission(Android.Manifest.Permission.AccessFineLocation)]
[assembly: UsesPermission(Android.Manifest.Permission.AccessCoarseLocation)]
````

**Comment by anttikajanus** [comment]
@apurvag123 Could you tackle this issue?

**Comment by anttikajanus** [comment]
https://github.com/Esri/arcgis-runtime-samples-xamarin/issues/113
https://github.com/Esri/arcgis-runtime-samples-xamarin/issues/112

**Comment by anttikajanus** [comment]
https://github.com/Esri/arcgis-runtime-samples-xamarin/issues/114

**Comment by apurvag123** [comment]
@anttikajanus 
I added the keys to info.plist for the display device location sample. So once that pull request is merged then all iOS samples will have location enabled.

For Android do u just want me to add it to manifest on master branch?

**Comment by anttikajanus** [comment]
@apurvag123 please also do the changes to Forms too and submit separate pull request for it.

**Comment by apurvag123** [comment]
I added location preferences to Android manifest in Xamarin Android project

**Comment by apurvag123** [comment]
merged
